internal/driver/gomobile/driver.go

```diff
@@ -11,6 +12,7 @@ import (
 	"golang.org/x/mobile/gl"
 
 	"fyne.io/fyne"
+	util "fyne.io/fyne/internal/driver"
```

Comment:
Why is the driver package renamed here?

Reply:
it was because the local type is "driver" - but that can be fixed